src/website/src/pages/ec/components/splash-page/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Splash page
+defaultTab: usage
+status: ready
+---
```

Comment:
i'm not sure the reason why we add, but only sometimes, the "tab" linking to storybook. i find it handy if you want to save a click when you know you want to reach the playground.

Reply:
I agree, we should have it for every components. I'll add a small task for that